checkstyle.xml

```diff
@@ -231,21 +231,21 @@
              value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
         </module>
         <module name="JavadocMethod">
-            <property name="scope" value="public"/>
```

Comment:
I believe we only need to change the scope to ignore JavaDoc on methods. Can you try changing the scope from 
`public` to `nothing`?
```
 <property name="scope" value="nothing"/>
```

Reply:
sure. Thanks. :D

Reply:
I'd be easier (I think) if you reverted the values you introduced and only modify the scope?